### PR TITLE
Scala 2.12.0-RC2

### PR DESCRIPTION
**Pull request by larsrh**

### CODE REVIEW

**Comment by larsrh** [comment]
Postponing this; blocked by a release of `scalajs-dom`, and possibly #143.

**Comment by larsrh** [comment]
Released `0.6.2` based on 11b85fa.

**Comment by larsrh** [comment]
Note to self: send PR to Scala community build to use upstream scalatags after this is merged. (See also scala/community-builds#371.)

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=0.13.8
+sbt.version=0.13.12
```
Comment (SethTisue) [inline project/build.properties]: could go to 0.13.13 now

File: .travis.yml [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
-- sbt ++2.11.5 readme/run
-- sbt ++$TRAVIS_SCALA_VERSION scalatagsJVM/test scalatagsJS/test
+  - sbt ++2.11.8 readme/run
+  - sbt ++$TRAVIS_SCALA_VERSION scalatagsJVM/test scalatagsJS/test
 scala:
-  - 2.10.4
-  - 2.11.4
+  - 2.10.6
   - 2.11.8
+  - 2.12.0-RC2
```
Comment (SethTisue) [inline .travis.yml]: 2.12.0 is on Maven Central now

Comment (larsrh) [inline .travis.yml]: I'm on it :smile:

File: project/build.properties (hunk 1 above)
Comment (larsrh) [inline project/build.properties]: 8b7dbab

**Comment by larsrh** [comment]
`0.6.2` for 2.12 is out.

**Comment by lihaoyi** [comment]
Looks good

**Pull request merged**